build.gradle.kts

```diff
@@ -112,7 +119,10 @@ val lintMarkdown by
 
 tasks.named("check") { dependsOn("buildHealth", lintMarkdown) }
 
-tasks.named("shellcheck") { group = "verification" }
+tasks.named("shellcheck") {
+  group = "verification"
+  dependsOn("nodeSetup")
```

Comment:
How is `shellcheck` related to the node.js installation?  I thought `shellcheck` was implemented in Haskell?